GripOpGras2.Server/appsettings.json

```diff
@@ -6,6 +6,6 @@
     }
   },
   "AllowedHosts": "*",
-  "FarmMapsApiBaseUrl": "https://farmmaps.eu/",
-  "FarmMapsAccountBaseUrl": "https://accounts.farmmaps.eu/"
```

Comment:
In task zetten dat dit in de toekomst afhankelijk van de dev of release build de niet testversie moet zijn.